config/locales/en.yml

```diff
@@ -62,7 +62,7 @@ en:
       start: Get started
       verify_email: Verify your email address
     totp_setup:
-      start: Set up two-factor authentication app
+      start: Set up two-factor authentication
```

Comment:
This is specifically for setting up the App.   I suggest we call it `Authenticator App`.